tests/model/templates/team-response-template.constants.ts

```diff
@@ -0,0 +1,8 @@
+export const EXPECTED_TEAM_RESPONSE_HEADER = "# Team's Response";
+export const EXPECTED_TEAM_RESPONSE_TEMPLATE_CONTENT = 'This is a simple response';
+
+export const DUPLICATE_ISSUE_NUMBER = 100;
```

Comment:
I feel that the tests here are quite simple; we can actually move the test data into the same file as the test cases.

Reply:
Regarding the test cases, I tried including additional test cases to cover the parseError field but did not manage to hit it. What I did was to try test cases with the wrong header format, and with the wrong format for duplicateOf. The outcome was that the teamResponse field on the template is undefined instead of the parseError field getting populated. Not sure if this is due to wrong test input used?

Testing wise for the teamResponse regex was done with whitespaces, are there any other combinations that can be used for the regex testing?

Reply:
Thanks for trying this.
What were the wrong formats that you tried?
I think these test cases are fine for now. We can add in the negative tests when we design proper error handling.

Reply:
I tried having the wrong header so something like `$123` for the duplicate issue number of duplicateOf field. But yeap would be good to add negative tests next time